cmd/opentelemetry/cmd/all-in-one/main.go

```diff
@@ -198,7 +198,10 @@ func startQuery(v *viper.Viper, logger *zap.Logger, exporter configmodels.Export
 		*queryServiceOptions)
 
 	tracerCloser := initTracer(logger)
-	server := queryApp.NewServer(logger, queryService, queryOpts, opentracing.GlobalTracer())
+	server, err := queryApp.NewServer(logger, queryService, queryOpts, opentracing.GlobalTracer())
+	if err != nil {
+		logger.Fatal("Could not create jaeger-query service", zap.Error(err))
```

Comment:
returning the error is more consistent in this function:
```suggestion
		return nil, nil, fmt.Errorf("Could not create jaeger-query service: %w", err)
```